src/utils/getLatestDate.ts

```diff
@@ -0,0 +1,33 @@
+const getCurrentDate = (): string => {
```

Comment:
don't see test written for it

Reply:
Yep it will go in next PR as already there are too many files in this one